samples/snippets/README.rst

```diff
@@ -1,328 +0,0 @@
-.. This file is automatically generated. Do not edit this file directly.
```

Comment:
Could you add a note here instead that helps add a note to find the samples to https://github.com/GoogleCloudPlatform/python-docs-samples/tree/main/speech/